crates/vm/levm/tests/tests.rs

```diff
@@ -4298,7 +4298,7 @@ fn extcodesize_existing_account() {
 fn extcodesize_non_existing_account() {
     // EVM Playground: https://www.evm.codes/playground?fork=cancun&unit=Wei&codeType=Mnemonic&code='PUSH20%200x42%5CnEXTCODESIZE%5CnSTOP'_
     let operations = [
-        Operation::Push((20, "0x42".into())),
+        Operation::Push((20, U256::from(0xA))),
```

Comment:
0xA is the address of a precompile. We can use other address for avoiding issues with that.

Reply:
Thx Jere, done [3a4f931](https://github.com/lambdaclass/ethrex/pull/1503/commits/3a4f931247e94d3e99fcd97a075f3332c0a5a8c7)